libs/cde-visualization/src/lib/components/violin/violin.component.html

```diff
@@ -0,0 +1,46 @@
+<span class="mat-caption violin-label">
+  Violin
```

Comment:
Update to label and tooltip match spec should be "Violin Graph"
![image](https://github.com/user-attachments/assets/ddca1b3e-8192-42f2-a75c-834214e33619)